### PR TITLE
Remove local copy of jar libraries

### PR DESCRIPTION
**Pull request by est31**
Use their maven versions instead.

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -43,12 +44,9 @@ dependencies {
     }
     compile 'de.hdodenhof:circleimageview:1.2.2'
 
-/*  TODO: Enable this when the Android variant of the Gerrit Rest Java Client is officially released
-       The corresponding jar can then be removed
-    compile('com.urswolfer.gerrit.client.rest:gerrit-rest-java-client:0.8.6-android') {
+    compile('com.urswolfer.gerrit.client.rest:gerrit-rest-java-client:0.8.6') {
```
Comment (p4r4n01d) [inline app/build.gradle]: Please see the comment above (which you have removed also) as to why I am unhappy with this. The Android branch is still not ready and we cannot use the master branch as it conflicts with Android's deprecated version of Apache. See the Android section of the readme here: https://github.com/uwolfer/gerrit-rest-java-client

I am happy for you to build the jar and switch to it with the latest commits (here: https://github.com/uwolfer/gerrit-rest-java-client/pull/26) provided:
1. All the tests pass for the gerrit-rest-java-client project in the Android branch
2. You have tested all the processors in mGerrit with this and nothing is broken (I would expect all of them to work or none)

Comment (est31) [inline app/build.gradle]: So, to clarify:

1. The httpclient-android branch uses the outdated android version of httpclient

1. The jar currently in app/libs is built fom the android branch

right? But why is then httpclient-android version 4.3.5.1 listed as dependency below? Its the up to date version of the httpclient library, right? That would be the workaround as stated in the readme.

Comment (est31) [inline app/build.gradle]: Anyway, I've tried with the latest commit from httpclient-android, and it seems to not work for all repos. Can you remember which commit the current jar bases on? then I could try replacing it with a jitpack build of that commit.

Comment (p4r4n01d) [inline app/build.gradle]: I think I built it from the latest commit at (my repo):
https://github.com/p4r4n01d/gerrit-rest-java-client/tree/httpclient-android

compile 'org.apache.httpcomponents:httpclient-android:4.3.5.1' is listed
​ separately in build.gradle as ​it was not being included when I imported
the jar so I needed to list each dependency seperately - which is why guava
is there (it is only used for the Gerrit http client library.
The httpclient-android branch
​DOES NOT ​
use the outdated android version of httpclient
​ it uses the new http client (for most of the functions) and a newer
version of the http client to override those used in Android. I cannot
remember why I needed both exactly, it just was that way.

Note to self: Seriously need to clarify this more in comments...​

Comment (est31) [inline app/build.gradle]: Mhhh, it seems that that branch can't be built with jitpack. There is a fix [waiting for be merged in master](https://github.com/uwolfer/gerrit-rest-java-client/pull/28) however.

I can't really open all the repos with the gerrit jar provided either, so I don't know whether the current upstream build at https://github.com/uwolfer/gerrit-rest-java-client/tree/httpclient-android is worse. Most repos work, except for a few. Can you open all repos? How long does it take until something shows up? More than ten secs?

**Comment by p4r4n01d** [comment]
Sorry for the late reply, I have been busy working on the next feature. 
I have updated the Gerrit API library (available here: https://github.com/p4r4n01d/gerrit-rest-java-client/tree/httpclient-android), fixed an old bug around starring changes and packaged into a jar with maven (local install). This automatically ensures all the tests pass (not sure why the bug wasn't picked up).
I am happy to merge this, however I don't want to take the changes to the Gerrit API library. I am going to leave it as an included jar due to the problems mentioned above with Apache and so Android-specific bugfixes can be included without impacting the main non-Android branch.
I can include this in the next release, along with a number of other bugfixes and the refine search feature.

**Comment by est31** [comment]
Okay, see #42

**Pull request closed without merging**